go.mod

```diff
@@ -5,6 +5,8 @@ go 1.15
 require (
 	github.com/google/go-cmp v0.5.6
 	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/otel/sdk v1.0.0-RC2
+	go.opentelemetry.io/otel/sdk/metric v0.0.0-00010101000000-000000000000
```

Comment:
Sorry, do you know why this sdk/metric module needs to be added to the require dependencies of go.mod? I don't think that would be right, since then the stable top-level "go.opentelemetry.io/otel" (in module set stable-v1) would have a dependency on the unstable module `go.opentelemetry.io/otel/sdk/metric`. Or is it just a placeholder for something?

Reply:
(Agree, this looks incorrect.)